### ISSUE TITLE
Floating point exception in exact Rmin computation

### ISSUE THREAD

**Issue by kleinj** [moves-rwth/storm#13]
When running
```bin/storm --prism robot.nm --prop 'Rmin=? [ F "goal2" ]' --exact```
for [functionality/verify/mdps/rewards/robot.nm](https://raw.githubusercontent.com/prismmodelchecker/prism-tests/master/functionality/verify/mdps/rewards/robot.nm) from the PRISM test suite, I get a floating-point exception:
```
Model checking property R[exp]min=? [F "goal2"] ...
 WARN (MinMaxLinearEquationSolver.cpp:161): Selecting policy iteration as the solution method to guarantee exact results. If you want to override this, please explicitly specify a different method.
 WARN (SparseMdpPrctlHelper.cpp:407): Results of reward computation may be too low, because of zero-reward loops.
Floating point exception
```

Looks like an unhandled division-by-zero in the SparseLU processing.

Might be related to the handling of improper schedulers in Rmin policy iteration?

**Comment by kleinj** [comment]
I noticed some other (perhaps related) test case with a floating point exception, with a Pmax:
```
bin/storm --prism ./example.nm --prop 'Pmax=? [ F s=2 ]' --exact
```
for [example.nm](https://raw.githubusercontent.com/prismmodelchecker/prism-tests/master/example.nm).

Interestingly, for this model / property, non-exact policy iteration
```
bin/storm --prism ./example.nm --prop 'Pmax=? [ F s=2 ]' --method pi --verbose
```
does not converge:
```
Model checking property Pmax=? [F (s = 2)] ...
 INFO (SparseMdpPrctlHelper.cpp:125): Found 1 'no' states.
 INFO (SparseMdpPrctlHelper.cpp:126): Found 1 'yes' states.
 INFO (SparseMdpPrctlHelper.cpp:127): Found 2 'maybe' states.
 INFO (GmmxxLinearEquationSolver.cpp:142): Solving linear equation system (2 rows) with Gmmxx linear equation solver with method 'GMRES' and preconditioner 'ilu' (max. 20000 iterations).
 WARN (GmmxxLinearEquationSolver.cpp:203): Iterative solver did not converge.
 INFO (IterativeMinMaxLinearEquationSolver.cpp:421): Iterative solver converged after 1 iterations.
 INFO (model-handling.h:456): Result (for initial states):
Result (for initial states):  INFO (model-handling.h:398): -nan
```

Might also be an issue of the initial scheduler?

**Comment by cdehnert** [comment]
Fixed as of commit cffc3e606e152edec8e0d154def7130882662cdf.
